### ISSUE TITLE
Can't get it to work

### ISSUE THREAD

**Issue by Burathar** [sukima/xmledit#53]
Hey guys,
I'm having a problem getting this plugin to work properly...
I started off by just copying the ftplugin files manually, but that didn't seem to work. I cleared my .vimrc, I removed my other ftplugin files, but it didn't help. Then I tried the .vba method, but no success.. after some trying I found out that the jumping from the opening tag to the closing tag using <leader>% _did_ work, but the automatic closing tags don't.
I am right that it should be possible to run `vim test.xml`, and then type `<xml>` and it should add `</xml>`? Or did I misunderstand something?

I'm running VIM 8.1.1401 on Debian GNU/Linux 10.4.

I hope someone can point out where I'm going wrong!

**Comment by albertdev** [comment]
Hi,

It is of course hard to troubleshoot this issue when it is not clear what is causing it. Jumping between the closing tags is supposed to happen when you press `<localleader>%` and <kbd>%</kbd> on its own. (You can check which key `<localleader>` is bound to by running `:let maplocalleader` )

First of all, are you sure you have placed the ftplugin script in the right directory? If you want plugins to be loaded then you obviously need to make sure that the `runtimepath` setting points to their parent directory (i.e. the directory containing the `ftplugin` directory)

Easiest way to check that setting:

* Open an empty file or new buffer
* Enter Insert mode.
* Press <kbd>Ctrl</kbd>+<kbd>R</kbd>, then press <kbd>=</kbd>. You will get an `=` prompt in the commandline area.
* Enter `&runtimepath`. Press <kbd>Enter</kbd>
* Now run `:%s/,/\r/g` to format

If some directory is missing, check your configuration. If you're using `pathogen` or the Vim package manager then maybe you forgot to declare the parent directory.

If the parent directory __is__ there then you might need to double check if things are getting loaded in the right order. You can troubleshoot that using this:

```
:redir @a
:scriptnames
:redir END
:new
"aP
```

This will give you a file in which you can look up what scripts have been loaded. This plugin's `xml.vim` should be in there and be loaded before the one built into Vim.

**Comment by Burathar** [comment]
Hi @albertdev ,
Thank you for your response. Firstly, you are right about the <localleader>%, this is what I was doing. Unfortunately I forgot this while writing the issue.. 
Furthermore; I found the problem. I would swear I tried this before, but apparently I didn't. The ftplugin directory was in my .vim/after dir instead in .vim itself. Moving it there fixed the problem. I'm sorry for bothering you with it..
